internal/cli/root.go

```diff
@@ -86,6 +86,7 @@ func buildRootCmd(cli *cli) *cobra.Command {
 		PersistentPreRunE: func(cmd *cobra.Command, args []string) error {
 			ansi.Initialize(cli.noColor)
 			prepareInteractivity(cmd)
+			cli.configureRenderer()
```

Comment:
This method mostly serves to attach the active tenant to the renderer struct so that we can make the comparison below. Otherwise, the tenant commands' renderers do not have knowledge of the currently active tenant.